modules/service/rest-api-v2.js

```diff
@@ -116,6 +116,11 @@ class RestAPIServiceV2 {
             await this._checkForHandlerStatus(req, res);
         });
 
+        server.post(`/api/${this.version_id}/challenges`, async (req, res) => {
```

Comment:
This is a get function

Reply:
it accepts start and end date, that's the reason why it's post